### PR TITLE
chore: add connection name to breadcrumbs COMPASS-7684

### PR DESCRIPTION
**Pull request by kmruiz**
### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by himanshusinghs — CHANGES_REQUESTED** [review]

**Comment by paula-stacho** [comment]
This is only touching the single Collection breadcrumbs, we also need to add breadcrumbs to Collections list.

File: packages/compass-collection/src/components/collection-header/collection-header.tsx [hunk 1]
```diff
@@ -98,13 +99,21 @@ export const CollectionHeader: React.FunctionComponent<
...
+  const connectionId = connectionInfo.id;
+  const connectionName = getConnectionTitle(connectionInfo);
 
   const breadcrumbItems = useMemo(() => {
     return [
-      // TODO (COMPASS-7684): add connection name
+      {
+        name: connectionName,
+        onClick: () => openDatabasesWorkspace(connectionId),
+      },
```
[the remark this answers is not in the document]
Comment (paula-stacho) [reply packages/compass-collection/src/components/collection-header/collection-header.tsx:116]: Same. The path is still valid, but feels kinda off if we've arrived through Databases menu item.

[the remark this answers is not in the document]
Comment (kmruiz) [reply packages/compass-collection/src/components/collection-header/collection-header.tsx:116]: That's the behaviour of the breadcrumb, clicking on the connection name opens the list of databases. This is not strictly related to multiple collections, but a task that Basit asked us to do in multiple connections, it's actually related to Data Explorer.

[the remark this answers is not in the document]
Comment (himanshusinghs) [reply packages/compass-collection/src/components/collection-header/collection-header.tsx:116]: I see. Ok that answers my question, thank you :)

**Review by himanshusinghs — APPROVED** [review]

**Pull request merged**